packages/compass-web/sandbox/index.tsx

```diff
@@ -27,6 +27,7 @@ import { LoggerAndTelemetryProvider } from '@mongodb-js/compass-logging/provider
 import { mongoLogId } from '@mongodb-js/compass-logging';
 import type { LoggerAndTelemetry } from '@mongodb-js/compass-logging';
 import type { MongoLogWriter } from 'mongodb-log-writer';
+import type { ConnectionInfo } from '@mongodb-js/connection-info';
```

Comment:
Can we re-export this from connection storage? Very similar to other providers, there is almost always the need to use the type that is returned by hooks / locators when they are used themselves, it's a bit inconvenient to have two deps instead of one when this happens

Reply:
yeah, will do that!